shell/platform/android/io/flutter/view/AccessibilityViewEmbedder.java

```diff
@@ -409,43 +413,65 @@ public int hashCode() {
     }
 
     private static class ReflectionAccessors {
-        private final Method getSourceNodeId;
-        private final Method getParentNodeId;
-        private final Method getRecordSourceNodeId;
-        private final Method getChildId;
-
+        private @Nullable final Method getSourceNodeId;
+        private @Nullable final Method getParentNodeId;
+        private @Nullable final Method getRecordSourceNodeId;
+        private @Nullable final Method getChildId;
+        private @Nullable final Field childNodesField;
```

Comment:
nit: consider calling this childNodeIdsField to better match the field name(on initial read I was expecting that you'll get a child "node" from this field and extract the id)